build.sbt

```diff
@@ -51,6 +52,8 @@ libraryDependencies ++= Seq(
   "com.typesafe.scala-logging" %% "scala-logging" % "3.9.2",
   "org.jboss.logging" % "jboss-logging" % "3.4.1.Final",
   "com.lightbend.akka" %% "akka-stream-alpakka-slick" % "1.1.2",
+  "org.scala-lang.modules" %% "scala-parser-combinators" % "1.1.2",
```

Comment:
Is this being used anywhere?

Reply:
It's not, I think it's a copy and paste error :sweat: I'll remove it :+1: